great_tables/_utils_render_html.py

```diff
@@ -6,9 +6,17 @@
 from htmltools import HTML, TagList, css, tags
 
 from . import _locations as loc
-from ._gt_data import GroupRowInfo, GTData, Styles
+from ._gt_data import (
+    ColInfo,
+    ColInfoTypeEnum,
+    GroupRowInfo,
+    GTData,
+    StyleInfo,
+    Styles,
+    SummaryRowInfo,
+)
 from ._spanners import spanners_print_matrix
-from ._tbl_data import _get_cell, cast_frame_to_string, replace_null_frame
+from ._tbl_data import TblData, _get_cell, cast_frame_to_string, replace_null_frame
```

Comment:
Let's put TblData in a if TYPE_CHECKING block, to flag that we are not instantiating any Tbly things